priv/repo/migrations/20210804999992_refresh_part_components.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Oli.Repo.Migrations.RefreshPartComponents do
+  use Ecto.Migration
+
+  def change do
+    alter table(:part_component_registrations) do
+      add :author, :string
+    end
+
+    flush()
+
+    execute "DELETE FROM part_component_registrations;"
```

Comment:
Just want to verify, the intent here is to delete all records from this table?

Reply:
yeah because they are populated by reading the manifest one time, and the updates are to the manifest values it seemed like the easiest way rather than coding something (or copying) the initial manifest code and turning it into a delta